open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminForViewServices.java

```diff
@@ -313,11 +315,44 @@ public VoidResponse configureAllViewServices(String userId,
     private ViewServiceConfig createViewServiceConfig(ViewServiceRegistration     registration,
                                                       ViewServiceConfig           requestedViewServiceConfig)
     {
-        ViewServiceConfig viewServiceConfig = new ViewServiceConfig(registration);
+
+        ViewServiceConfig viewServiceConfig = null;
+
+        if (requestedViewServiceConfig instanceof IntegrationViewServiceConfig) {
```

Comment:
I suggested in https://github.com/odpi/egeria/issues/3662 that we could have a type of the view service - that we could interrogate rather than using reflection.

Reply:
Perhaps - but it's another thing that the user has to set - I think I prefer it with just the class in the JSON

Reply:
I would not expect the user to set it - we use the name of the view service to get the registration details -where I would expect this flag to reside, most of the registration document does not need to sent in .